src/PayID/pay-id-client.ts

```diff
@@ -59,7 +60,7 @@ export default class PayIDClient {
    * @param payID The payID to resolve for an address.
    * @returns An address representing the given PayID.
    */
-  async addressForPayID(payID: string): Promise<string> {
+  async addressForPayID(payID: string): Promise<CryptoAddressDetails> {
```

Comment:
So `Xpring-SDK` only supports PayID for `CryptoAddressDetails`?

How would you retrieve `AchAddressDetails`?

Reply:
For now yes. At some point in the future we'll have to change this and return the complete wrapper and let the client fend for themselves. That feels like a worse API and we are already regularly making breaking changes here so it doesn't feel worth future proofing.

Reply:
I bet there are a limited number of `AddressDetailType`s that we'll support.

I wonder if eventually, we can just have a `addressForPayID` that has helper functions like `cryptoAddressForPayID` and `achAddressForPayID`, and it looks at the `addressDetailType` in the response payload to determine which helper method to use.